### DISCUSSION TITLE
Is anyone able to download the installer from the specified URL for Baidu.BaiduNetdisk？

### DISCUSSION THREAD

**Discussion by aaronliu0130** [microsoft/winget-pkgs#93814] [Q&A] [answered]
I haven't been able to download anything from issuepcdn, yet the pipeline checks for that pr have passed.

**Comment by Trenly** [comment, marked as answer]
Seems to be working for me; Just takes a while since it seems the download is quite slow
![image](https://user-images.githubusercontent.com/12611259/212593379-565caeeb-c478-4a3b-ad14-242dab9122e6.png)
